src/polyfill/array-fill.js

```diff
@@ -0,0 +1,72 @@
+if (!Array.prototype.fill) {
```

Comment:
```suggestion
// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill#polyfill
if (!Array.prototype.fill) {
```

Reply:
(I like to link back to where polyfills are sourced generally...)

Reply:
done